spec/models/miq_expression_spec.rb

```diff
@@ -308,6 +308,49 @@
     context "relative date/time support" do
       around { |example| Timecop.freeze("2011-01-11 17:30 UTC") { example.run } }
 
+      context "given a non-UTC timezone" do
+        it "generates the SQL for a = expression with a value of 'Yesterday' for a date field" do
```

Comment:
If I rebase this after https://github.com/ManageIQ/manageiq/pull/9960 I should be able to get this back to green